examples/Getting Started with Semiconductor Module/Getting Started with Semiconductor Module.py

```diff
@@ -0,0 +1,215 @@
+""" Getting Started with Semiconductor Module example.
+
+    Example NI TSM Test Program using simulated NI-Digital and NI-DCPower instruments
+"""
+
+import random
+
+import nidcpower
+import nidigital
+from nitsm.codemoduleapi import (
+    SemiconductorModuleContext,
+    code_module,
+    Capability,
+    InstrumentTypeIdConstants,
+)
+
+
+@code_module
+def open_dcpower_sessions(tsm_context: SemiconductorModuleContext):
+    """Open NI-DCPower instrument sessions."""
+    resource_strings = tsm_context.get_all_nidcpower_resource_strings()
+    for resource_string in resource_strings:
+        session = nidcpower.Session(
+            resource_string,
+            options={"Simulate": True, "DriverSetup": {"Model": "4143", "BoardType": "PXIe"}},
+        )
+        tsm_context.set_nidcpower_session(resource_string, session)
+
+
+@code_module
+def open_digital_sessions(tsm_context: SemiconductorModuleContext):
+    """Open NI-Digital instrument sessions."""
+    instrument_names = tsm_context.get_all_nidigital_instrument_names()
+    for instrument_name in instrument_names:
+        session = nidigital.Session(
+            instrument_name, options={"Simulate": True, "driver_setup": {"Model": "6570"}}
+        )
+        tsm_context.set_nidigital_session(instrument_name, session)
+
+
+@code_module
+def open_all_instruments_sessions(tsm_context: SemiconductorModuleContext):
+    """Open all instrument sessions."""
+    open_dcpower_sessions(tsm_context)
+    open_digital_sessions(tsm_context)
+
+
+@code_module
+def continuity(
+    tsm_context: SemiconductorModuleContext,
+    pins,
+):
+    """Measure continuity on configured Pins (NI-DCPower, NI-Digital).
+
+    This example uses simulated data and driver calls so it can run on a system without the
+    NI-Digital Pattern or NI-DCPower drivers installed. Refer to the Accelerometer example
+    for an example of a continuity test using the these drivers.
+    """
+    dcpower_filtered_pins = tsm_context.filter_pins_by_instrument_type(
+        pins, InstrumentTypeIdConstants.NI_DCPOWER, Capability.ALL
+    )
+    pin_query, sessions, channel_strings = tsm_context.pins_to_nidcpower_sessions(
+        dcpower_filtered_pins
+    )
+    per_dcpower_session_measurements = []
+    dcpower_continuity_measurements = []
+    for session, channel_string in zip(sessions, channel_strings):
+        # simulate continuity data for each channel
+        per_dcpower_session_measurements = []
+        for _ in channel_string.split(","):
+            per_dcpower_session_measurements += simulate_parametric_measurement(0.75, -0.75, 0.17)
```

Comment:
let's just use a `simulate_continuity_data` function without arguments like in the LV example